code/modules/medical/cloning.dm

```diff
@@ -26,6 +26,7 @@
 	var/time_coeff = 1 //Upgraded via part upgrading
 	var/resource_efficiency = 1
 	var/id_tag = "clone_pod"
+	var/super = 0 //if upgraded enough, it will go super and unlock a cool thing
```

Comment:
can you come up with a better variable name? `super` is really bad. I suggest `perfect_clones` if that's what your PR does

Reply:
literally just make it `perfect_clones`